brownie/network/rpc.py

```diff
@@ -372,3 +387,38 @@ def _check_connections(proc: psutil.Process, laddr: Tuple) -> bool:
         return laddr in [i.laddr for i in proc.connections()]
     except psutil.AccessDenied:
         return False
+
+
+def _validate_cmd_settings(cmd_settings: dict) -> dict:
+    CMD_TYPES = {
+        "port": int,
+        "gas_limit": int,
+        "block_time": int,
+        "time": datetime.datetime,
+        "accounts": int,
+        "evm_version": str,
+        "mnemonic": str,
+        "account_keys_path": str,
+        "fork": str,
+    }
+    for cmd, value in cmd_settings.items():
+        if (
+            cmd in CLI_FLAGS.keys()
+            and cmd in CMD_TYPES.keys()
+            and not type(value) == CMD_TYPES[cmd]
```

Comment:
can this be done using `isinstance()` instead of `type()`?

Reply:
yes :)